scripts/azure-pipelines/build.ps1

```diff
@@ -0,0 +1,48 @@
+﻿# Builds Mumble using the given environment and build script.
+# The configuration we build with is adjusted to be close to
+# our release builds.
+#
+# Configuration variables used from environment:
+#
+#  AGENT_BUILDDIRECTORY       - The local path on the agent where all folders
+#                               for a given build pipeline are created
+#                               (e.g. "D:\a\1")
+#  BUILD_SOURCESDIRECTORY     - The local path on the agent where the
+#                               repository is downloaded.
+#                               (e.g. "D:\a\1\s")
+#  MUMBLE_ENVIRONMENT_DIR     - The local path where the build environment
+#                               will be stored (e.g. "C:\MumbleBuild").
```

Comment:
"will"? Isn't this the assumption/requirement this script makes?

Reply:
Indeed, I copied the description from `install-environment.ps1` and didn't change "will be" to "is".